DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Administrator/_SearchableAdminCard.cshtml

```diff
@@ -53,7 +53,7 @@
           Manage roles
         </a>
       }
-      @if (!Model.IsCentreManager) { 
+      @if (Model.HasSuperAdminAccess && Model.CurrentAdminUserId != Model.Id)  { 

```

Comment:
This is the only place these are used, so we can simplify the logic in the view to a simple check of a 'should show deactivate admin button' property on the model, and set that property with these values in the model constructor. This minimises the logic in the cshtml file and lets us add a quick test for the behaviour.

Reply:
I have simplified the check as suggested. Thanks

Reply:
It looks great, but could we add a quick test to verify this behaviour?

Reply:
I have added tests for this behaviour